Controller/ReceivedUrlTrait.php

```diff
@@ -65,6 +65,7 @@ public function execute()
         $boltPayload = $this->getRequest()->getParam('bolt_payload');
         $storeId = $this->getRequest()->getParam('store_id');
 
+        // phpcs:ignore
```

Comment:
2 ignores here in this file - we don't have a good alternative to `base64_decode` right now and we're not using it in a particularly unsafe manner.